modules/data-entry/src/main/frontend/src/vocabQuery/VocabularyQuery.jsx

```diff
@@ -266,9 +248,36 @@ function VocabularyQuery(props) {
           </MenuItem>
           );
       });
-      showUserEntry && suggestions.push(<Divider key="divider"/>);
     }
+
+    var allRequestsFailed = Object.keys(statuses).filter(vocab => !statuses[vocab]).length == 0;
+    var allRequestsSucceded = Object.keys(statuses).filter(vocab => statuses[vocab]).length == 0;
+
+    if (!allRequestsSucceded) {
+      suggestions.push(
```

Comment:
I would add another divider between the error message and any existing suggestions:

```suggestion
    if (!allRequestsSucceded) {
      if (!allRequestsFailed) {
        suggestions.push(<Divider key="error-divider"/>);
      }
      suggestions.push(
```

Reply:
Shouldn't we add `suggestions.length > 0 &&` here?

Reply:
Yes we should